resolwe_bio/utils/test.py

```diff
@@ -93,6 +99,19 @@ def prepare_expression(self, f_rc='exp_1_rc.tab.gz',
         return self.run_process('upload-expression', inputs)
 
 
+class BioProcessTestCase(TransactionBioProcessTestCase, DjangoTestCase):
```

Comment:
Do we even need transaction and non-transaction tests cases? What are advantages of having two instead of just one (non-transaction)?